Tools/AP_Bootloader/board_types.txt

```diff
@@ -359,6 +359,9 @@ AP_HW_ZeroOne_X6                     5600
 
 # IDs 6600-6699 reserved for Eagle Eye Drones
 
+# IDs 7000-7099 reserved for CUAV
```

Comment:
Any particular reason for the huge gap you're leaving?

Reply:
No, just think the number looks good. Thank you for merging.